src/avkit.cs

```diff
@@ -323,7 +299,7 @@ interface AVPlayerViewControllerAnimationCoordinator {
 	}
 
 #else
-
+	
```

Comment:
Minor but careful with whitespace changes (please do not commit them).

A nice trick to avoid that is when you need to stage your changes before committing, try to use `git add -i`. Then you do (5 to patch), select all your files (1,2,3...) and that will have you review every single line of code you need to stage (you can use `s` to split if needed).
So you go line by line and press (y/n) and can easily ignore the whitespace changes this way.
You'll end up with staged and unstaged changes (all unstaged would be the things you don't want to commit). Then you can `git commit` and `git reset --hard` all the unstaged / unwanted code.

Reply:
That's a nice trick right there 😄 thanks